data/mods/Magiclysm/worldgen/orc_village.json

```
@@ -0,0 +1,38 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+  "111111111111111111111111111111111111111111111111111111111",
+  "1.......................................................1",
+  "1.......................................................3",
+  "1.......................................................1",
+  "1.......................................................3",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "1.......................................................1",
+  "111111111111111111111111111111111111111111111111111111111",
+  "                                                         " ],
+        "place_monsters": [
```

```suggestion
        },
        "place_monsters": [
```
Closes the "object:" above.